lib/cli/interactive-setup/utils.js

```diff
@@ -35,4 +38,12 @@ module.exports = {
 
     return Boolean(result);
   },
+  resolvePreliminaryContext: ({ configuration, serviceDir }) => {
+    return {
+      isInServiceContext: Boolean(serviceDir),
+      isLoggedIn: isAuthenticated(),
+      hasLocalCredentials: hasLocalCredentials(),
```

Comment:
Let's maybe name it `hasLocalAwsCredentials`

Reply:
Agreed, it's much better name :+1: